drivers/hmis/app/models/hmis/hud/processors/client_processor.rb

```diff
@@ -75,6 +75,24 @@ def schema
     def information_date(_)
     end
 
+    def post_process
+      # RaceNone and GenderNone should be set to 99 (Data Not Collected) if no Race/Gender fields are 1.
+      # input_to_multi_fields handles this when data is submitted as an empty list, but
+      # if race/gender data is submitted as individual fields ('Client.Woman' => '0', 'Client.Transgender' => '0')
+      # then we need to update RaceNone/GenderNone to 99 here in post-processing.
+      client = @processor.send(factory_name)
+
+      race_fields = HudUtility2024.races.keys.excluding('RaceNone')
+      any_race = race_fields.any? { |field| client.send(field) == 1 }
```

Comment:
I believe this can be written as 
```suggestion
      any_race = client.race_fields.excluding('RaceNone').any?
```

Using the helper from this concern (shared with wh and hmis client models) https://github.com/greenriver/hmis-warehouse/blob/44113f9cba11910aff1f64b9918ccb472b463dae/app/models/concerns/hud_concerns/client.rb#L16-L24

Reply:
Yes looks good, thanks!